app/controllers/comments_controller.rb

```diff
@@ -34,7 +34,7 @@ def index
       set_comments_made
     else
       @page_title = "Recent Comments"
-      @pagy, @comments = pagy(Comment.on_track.includes(commenter: :pic, commentable: { user: :pic }).public_or_private(moderator?))
+      @pagy, @comments = pagy(Comment.includes(commenter: :pic, commentable: { user: :pic }).public_or_private(moderator?))
```

Comment:
We used to have comments on multiple other types of records. So removing `on_track` will potentially surface some errors on older data.